### PR TITLE
Add basic Header component

### LINKED ISSUE #3: Requests list Header
Screen Shot 2020-03-29 at 11 39 15 PM

### PR DESCRIPTION
**Pull request by StanleySathler**
Closes #3

It adds a basic Header component, containing static data. We can add props and any other features on demand.

Screenshot_2020-04-03_14-36-08

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { ReactComponent as RequestIcon } from '../icons/request.svg';
+import styles from './Header.module.css';
+
+function Header() {
+  return (
+    <header className={styles.header}>
+      <div className={styles.iconBox}>
+        <RequestIcon />
+      </div>
+      <h2>Requests</h2>
```
Comment (dielduarte) [inline src/components/Header.tsx:11]:
```suggestion
      <h1>Requests</h1>
```

let's change to h1 just due to semantic reasons, this is kinda the principal title of the page and will change as you can see here: https://github.com/dielduarte/graphql-devtools/issues/6#issuecomment-607613212. I think you can just accept the suggestion, thanks!

**Comment by dielduarte** [comment]
@StanleySathler just to you know, it breaks the copy icon position in the CodeEditor, but I think we can fix it while working in the task to refactor the code editor (https://github.com/dielduarte/graphql-devtools/issues/1)


Screen Shot 2020-04-04 at 2 09 43 AM

thank you!

File: src/components/Header.tsx (hunk 1 above)
Comment (dielduarte) [reply src/components/Header.tsx:11]: gonna merge it by myself, cause I will need Header to work on settings module @StanleySathler !

Comment (StanleySathler) [reply src/components/Header.tsx]: No worries, buddy :rocket: Well noticed!

**Pull request merged**